src/frontend/packages/eslint-config-marsha/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "eslint-config-marsha",
+  "version": "0.0.0",
```

Comment:
we should use the same version used in all other packages.

Reply:
https://github.com/openfun/marsha/pull/2150/commits/921e29cffc24e8eae16d2c07f538923bfc9ed3cf
https://github.com/openfun/marsha/pull/2150/commits/7a2558a947bc1ef02a8b733de339775b9eb6e1d8